main.go

```diff
@@ -48,6 +49,7 @@ var (
 	gkeNamespace               = flag.String("gke-namespace-experimental", "", "GKE namespace to use. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	gceVM                      = flag.String("gce-vm-experimental", "", "GCE VM name to use, instead of reading it from the metadata server. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	configMesh                 = flag.String("config-mesh-experimental", "", "Dictates which Mesh resource to use. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
+	generateMeshId             = flag.Bool("generate-mesh-id", false, "whether or not to generate the mesh ID as per CSM GAMMA Mesh manual hashing algorithm. Usage of this flag requires gke-cluster-name-experimental to be set and config-mesh-experimental to be unset. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
```
gke-cluster-name-experimental to be set
```

Can't we get this information from the metadata server? This can be resolved in a follow-up PR.

```
and config-mesh-experimental to be unset
```

We meed to enforce this requirement.

Reply:
I think the help text here should be a little more friendly. Maybe instead of "whether or not to generate the mesh ID as per CSM GAMMA Mesh manual hashing algorithm.", "When enabled, enrolls the bootstrap in a CSM GAMMA mesh for the indicated GKE cluster."

Reply:
Done. I've updated per your comments.